cli/packages/visualize/dynamic_secret_leases.go

```diff
@@ -0,0 +1,38 @@
+package visualize
```

Comment:
The file is called `dynamic_secret`, but you call the command with `dynamic-secrets`. Can you please make the file name align with the command name?

Reply:
Sorry commented on the wrong file it seems. The `cmd/dynamic_secret.go` file should be called `cmd/dynamic_secrets.go` so it matches the actual command name (`infisical dynamic-secrets [...]`)

Reply:
Ahh got it

Reply:
Updated